lib/mv_opentelemetry.ex

```diff
@@ -53,6 +53,8 @@ defmodule MvOpentelemetry do
     live_view twice.
     - `default_attributes` OPTIONAL property list of attributes you want to attach to all traces
       from this group, for example [{"service.component", "my_app"}]. Defaults to []
+    - `query_params_whitelist` OPTIONAL list of query param names you want to allow to log in your
+      traces, i.e ["user_id", "product_id"]. Defaults to logging all.
```

Comment:
The reason for this slightly weird default is semantic versioning - we would want the original behaviour to be preserved.